### ISSUE TITLE
"File not found" message on the local server.

### ISSUE THREAD

**Issue by sruloart** [lucamezzalira/haxe-watchify#12]
Probably it's my own incompetence, but I can't get the path right. I've tried: 
http://localhost:35729/html5/bin/index.html
http://localhost:35729/html5/bin/#
(and even http://localhost:35729/html5/index.html, I've no idea why)

The server is running, I just can't see any files.

Also, in order to get Flash on an index.html I need to create the .html page myself first? it doesn't really matter because I can't seem to get any path right, but it's important to know. 

This is my hx-watch.json: 
```json
{
  "build": {
    "program" : "openfl",
    "buildType" : "test", 
	"src" : "source",
    "livereload" : "export",
    "platforms" : ["html5", "flash"]
  }
}
```

**Comment by lucamezzalira** [comment]
So let's start from the bottom:

. live reload with Flash: you've to create your HTML page in order to see the changes, probably if you point your browser to the swf file path should work as well but I think it's more valuable having an HTML page that you can relay on

* live reload path: can you share with a print screen the structure of export folder please? so I can guide you to have the correct url :D

**Comment by sruloart** [comment]
Sure! 
![tree](https://cloud.githubusercontent.com/assets/5843924/8856415/6eb0f884-3173-11e5-8d00-bb84b8ade17b.png)

**Comment by lucamezzalira** [comment]
in theory you should be able to see the result with this URL _http://localhost:35729/html5/bin/index.html_

**Comment by sruloart** [comment]
In theory yes :) but I'm getting just "File not found"...

**Comment by lucamezzalira** [comment]
yep, if this is not a production project, can you send it to me so I'll be able to test in a real case scenario?

**Comment by sruloart** [comment]
I would gladly, but something tells me you don't use the haxeflixel git branches and it can take a while to [set these up] (http://haxeflixel.com/forum/?place=msg%2Fhaxeflixel%2Ff_4JB6shrWY%2FmbLN3msejaIJ).

I'll make a minimal OpenFL project, test it to see if it has the same problem (it should, the export structure is the same, with or without HaxeFlixel), and then send it to your mail in a zip/rar archive.

**Comment by sruloart** [comment]
Done. Hopefully.

**Comment by lucamezzalira** [comment]
ok I found the solution for this issue and #10 I'm going to publish in few minutes, then let me know if it works. On my emulator it was working

**Comment by sruloart** [comment]
It's working and it's Absolutely Freaking Beautiful!

**Comment by lucamezzalira** [comment]
Solved in version 1.2.1
